packages/Takeover/src/components/Content.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+import { spacer } from "@paprika/stylers/lib/helpers";
+import tokens from "@paprika/tokens/lib/tokens";
+
+const Content = styled.div`
+  margin: ${spacer(2)};
+  padding: ${spacer(2)};
+  background-color: ${tokens.color.white};
+  border: 1px solid ${tokens.border.color};
+  border-radius: 3px;
+  box-shadow: ${tokens.shadow};
+`;
+
+Content.componentType = "Takeover.Content";
```

Comment:
while there are some components using `componentType`, the latest agreement is to use from now and forward`displayName` instead to detect component.